http2_general/test_h2_headers.py

```diff
@@ -438,3 +438,68 @@ class HeadersSpacedCache(CurlTestBase):
 
     def test(self):
         CurlTestBase.run_deproxy_test(self, served_from_cache=True)
+
+class MissingDateServerWithBodyTest(tester.TempestaTest):
+    """
+    Test response without Date and Server headers, but with short body.
+    """
```

Comment:
Could you please elaborate in the comment what the test verifies, i.e. what's specific about not having Server and Date (they aren't required headers) and short body (I guess we already have tons of such tests).